docs/sources/alerting/configure-notifications/mute-timings.md

```diff
@@ -85,6 +85,6 @@ If you want to specify an exact duration, specify all the options. For example,
 [datasources/alertmanager]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/datasources/alertmanager"
 [datasources/alertmanager]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/connect-externally-hosted/data-sources/alertmanager"
 
-[fundamentals/alertmanager]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/alerting/fundamentals/alertmanager"
-[fundamentals/alertmanager]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/fundamentals/alertmanager"
+[intro-alertmanager]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/alerting/notifications/alertmanager"
+[intro-alertmanager]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/alerting-and-irm/alerting/notifications/alertmanager"
```

Comment:
These links 404 for me.

Reply:
fixed by https://github.com/grafana/grafana/pull/84248/commits/f5512cbf2816f8d08b22bfc7086aa39ce6e7c342